Client/Ecosia/L10N/String.swift

```diff
@@ -124,10 +124,9 @@ extension String {
         case websitesYouHave = "Websites you’ve recently visited will show up here"
         case noDownloadsYet = "No downloads yet"
         case whenYouDownloadFiles = "When you download files they will show up here"
-        case heyThereWantToPlant = "Hey there, want to help plant trees with me? If you get the Ecosia app, we will both be contributing extra trees to be planted!"
-        case downloadEcosiaOn = "(1) Download Ecosia on your iPhone for free (friend invites for Android coming soon!):"
-        case afterInstalling = "(2) After installing the app, click my invitation link so we plant an extra tree each:"
-        case letsGrowTogether = "Let's grow together"
+        case checkThisOut = "Check this out: Ecosia plants trees every time you search the web! 🌳\nJoin me and 20M+ others and start planting today."
```

Comment:
to be future proof the 20M+ should be taken from our statistics endpoint:
https://d2wfixp891z15b.cloudfront.net

Reply:
Thank you for your feedback @ecotopian , though this looks more like a feature request.

I encourage you to create a ticket for it, but I'm not exactly sure I see the value of it (extra layer of complexity for a number that **might** change far in the future...), and I will also recommend you to discuss first it with the _Copy Team_, as this is actually their domain.

Reply:
As we have a dynamic number  in the json

`  
      "name": "Active Users",
      "value": "20000000.00",
      "last_updated": "2019-12-09T10:49:00.000000Z"
`

why shouldn't we use it? We do it similarly in the intro for our trees planted. Complexity is simple and it recently changed from 15M+ to 20M+. So I see a lot of value from it and we'll be growing.